resources/kds-team.ex-procore/templates/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Data Services CZ
```

Comment:
Just a triviality, you would probably want to unify the license files with the one above.

Reply:
But  I wonder, why do you want to include the licenses here anyway?

Reply:
Wait, what? :) I lived under the impression the MIT license has to be there. In that case, I'll just remove it then.

Thank you for looking into this!!!

Reply:
This repository is just a technical infrastructure for delivering the templates. We can move it to some other place or even integrate it into the Developer Portal itself. It should not be a display window so I think it's not necessary. ;o)